spec/requests/users_spec.rb

```diff
@@ -2,6 +2,34 @@
 
 RSpec.describe 'Users', type: :request do
   describe 'GET /index' do
-    pending "add some examples (or delete) #{__FILE__}"
+    before do
+      get users_path
+    end
+
+    it 'returns correct status' do
+      expect(response.status).to eq(200)
+    end
+
+    it 'renders the correct template' do
+      expect(response).to render_template('users/index')
+    end
+  end
```

Comment:
- [ ] [REQUIRED] Good job so far! Spec file include tests that ensure the response body includes the correct placeholder text for `user_spec`. Example : `expect(response.body).to include("Welcome to user index page")`, please implement it 🙏🏼

Reply:
- [ ] As required by the previous reviewer and as per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/controller_specs_project.md#:~:text=template%20was%20rendered.-,If%20the%20response%20body%20includes%20correct%20placeholder%20text,-.), Please add test that ensures the response body includes the correct placeholder text. I can see that you added two tests for the `users show` below but you need to add it also for the `users index`.